README.md

```diff
@@ -1,70 +1,34 @@
+# Morhpheus Node Setup
 
-Morhpheus Node Setup
+## Prerequisites
 
+- Node.js: https://nodejs.org/en/download
 
+## Running development environment
 
-Windows OS Systems
```

Comment:
The existing README only mentions Windows when the instructions are similar for all operating systems. These changes generalize the README so it's apparent that Morpheus works for more than only Windows.